### PR TITLE
Better logging

### PR DESCRIPTION
**Pull request by tomrf1**
Logs are now sent to logstash

The log file will now have a line per 'event', formatted as json. All logs contain the user ID and timestamp. Events are:
- `results_ok` - the user's response to some search results
- `search_status` - status of a capi query
- `wit` - result of wit.ai parsing
- `suggest` - when user enters "suggest"

### CODE REVIEW

**Comment by tsop14** [comment]
> results_ok - the user's response to some search results

Are we not logging when the users says "show me more"?

it is nice to read yaml 👀

**Comment by tomrf1** [comment]
Yes that's covered by the `results_ok` event, which probably isn't a very good name. It looks like:

```
{"id":"131403100605008","timestamp":"2016-06-08T12:22:57+0000","event":"results_ok","data":{"ok":false}}
{"id":"131403100605008","timestamp":"2016-06-08T12:22:58+0000","event":"search_status","data":{"success":true,"query":"carrot soup","page":2}}
```

File: logstash.conf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+input {
+  file {
+    path => "/home/content-api/logs/out*.log"
+    type => "application"
+  }
+}
+filter {
+  if ([message] !~ "^{") {
```
Comment (cb372) [inline logstash.conf]: Logstash expert!

**Comment by cb372** [comment]
Looks good to me

File: logstash.conf (hunk 1 above)
Comment (tomrf1) [inline logstash.conf]: *stackoverflow.com expert

**Pull request merged**